src/Components/NearBy/Card/NearbyMap.js

```diff
@@ -1,13 +1,25 @@
 import React, { useEffect } from "react";
+import {
+  Link,
+  matchPath,
+  useHistory,
+  useRouteMatch,
+  useParams,
+  withRouter,
+} from "react-router-dom";
 import styled from "styled-components";
 
 const { kakao } = window;
 
-const NearbyMap = () => {
+const NearbyMap = (props) => {
+  const { targetRoomData } = props;
```

Comment:
아래처럼 바로 작성해줄 수 있습니다 :) 

```suggestion
const NearbyMap = ({targetRoomData}) => {
 
```